# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../avifile/xvid4-needs-libm.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- avifile-0.7-0.7.41/m4/xvid4.m4~	2004-02-18 21:55:17.000000000 +0100
+++ avifile-0.7-0.7.41/m4/xvid4.m4	2004-10-26 18:55:47.122867616 +0200
@@ -33,7 +33,7 @@
 	AC_CHECK_HEADER(xvid.h, have_xvid4=yes)
     fi
     CPPFLAGS=$ac_save_CPPFLAGS
-    XVID4_LIBS="$XVID4_LIBS -lxvidcore"
+    XVID4_LIBS="$XVID4_LIBS -lxvidcore -lm"
 
 dnl
 dnl Now check if the installed XviD is sufficiently new.
